architecture/evm/error_normalizer.go

```diff
@@ -299,6 +299,20 @@ func ExtractJsonRpcError(r *http.Response, nr *common.NormalizedResponse, jr *co
 					nil,
 				)
 			}
+			// Special case for envio's handling of "eth_getBlockReceipts" where they don't support object of
+			// type "BlockNumber" in the "blockHash" field.
+			if strings.Contains(msg, "invalid type: map, expected BlockNumber, 'latest', or 'earliest'") {
```

Comment:
```suggestion
			if strings.Contains(msg, "invalid type:") {
```
How about this to make it more generic invalid arg err?

Reply:
I also thought keeping them more specific is better. also not sure if all `invalid type` errors fall under the `retryable` category. needs to be investigated.